.github/workflows/ci_tests.yml

```diff
@@ -119,6 +126,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        # There seems to be an issue with module search in Python 3.11
+        # python-version: ["3.9", "3.10", "3.11"]
         python-version: ["3.9", "3.10"]
```

Comment:
Is there an issue for addressing the 3.11 problems?

Reply:
Added issue #168